float64[3] force_offset
float64[3] moment_offset
float64[3] link_offset_centroid
float64 link_offset_mass
